src/webapi/window.rs

```diff
@@ -3,6 +3,26 @@ use webapi::event_target::{IEventTarget, EventTarget};
 use webapi::window_or_worker::IWindowOrWorker;
 use webapi::storage::Storage;
 use webapi::location::Location;
+use webcore::try_from::TryInto;
+use webcore::serialization::Once;
+use webcore::value::Value;
+
+/// A handle to a pending animation frame request.
+#[derive(Debug)]
+pub struct RequestAnimationFrameHandle(Value);
+
+impl RequestAnimationFrameHandle {
+    /// Cancels an animation frame request.
+    ///
+    /// [(Javascript docs)](https://developer.mozilla.org/en-US/docs/Web/API/Window/cancelAnimationFrame)
+    pub fn cancel(self) {
+        js!{
+            let val = @{self.0};
```

Comment:
Use `var` instead of `let`. (We support Emscripten, which doesn't like ES6+ features.)